### DISCUSSION TITLE
Use Vue libraries

### DISCUSSION THREAD

**Discussion by tabishiqbal** [vuejs/petite-vue#50] [Q&A]
Wanted to ask how you would use other libraries such as v-calendar.

**Comment by niutech** [comment]
Just convert them to [Petite Vue Components](https://github.com/vuejs/petite-vue#components-with-template), provided that they don't rely on [unsupported features](https://github.com/vuejs/petite-vue#not-supported). Sadly, V-Calendar is [too bloated](https://bundlephobia.com/package/v-calendar@2.3.2) for Petite Vue, use the full Vue.js instead.

But there is a lightweight [vuejs-datepicker](https://github.com/charliekassel/vuejs-datepicker), which I started porting to Petite Vue - you can play with the [demo](https://jsfiddle.net/x6sroy49/2/).
